### DISCUSSION TITLE
Is it normal when developing with Vite, the entire lib is loaded into the browser?

### DISCUSSION THREAD

**Discussion by planetaska** [svelteuidev/svelteui#178] [Q&A]
I tested in one of my project, and in a freshly created npm create app. Both of the projects uses yarn and Vite for bundling. When I import the Button component in development, I expect only the Button and necessary files to be loaded. But instead every single file in the SvelteUI lib (which is huge!) is loaded into the browser. The SvelteUI lib is so huge it takes a few seconds to load all things on every page refresh. Is this an expected and normal behavior, or is this a bug on the Vite or on the SvelteUI side?

**Comment by kamellperry** [comment]
This seems to be a bug on our side, as this behavior is fairly new. Thank you for pointing it out!
